src/test/kotlin/uk/gov/dluhc/printapi/service/temporarycertificate/TemporaryCertificateFinderServiceTest.kt

```diff
@@ -0,0 +1,62 @@
+package uk.gov.dluhc.printapi.service.temporarycertificate
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.any
+import org.mockito.kotlin.given
+import org.mockito.kotlin.verify
+import uk.gov.dluhc.printapi.database.entity.SourceType.VOTER_CARD
+import uk.gov.dluhc.printapi.database.repository.TemporaryCertificateRepository
+import uk.gov.dluhc.printapi.service.EroService
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidRandomEroId
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidSourceReference
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildTemporaryCertificate
+import uk.gov.dluhc.printapi.testsupport.testdata.getRandomGssCodeList
+
+@ExtendWith(MockitoExtension::class)
+internal class TemporaryCertificateFinderServiceTest {
+
+    @Mock
+    private lateinit var eroService: EroService
+
+    @Mock
+    private lateinit var temporaryCertificateRepository: TemporaryCertificateRepository
+
+    @InjectMocks
+    private lateinit var temporaryCertificateFinderService: TemporaryCertificateFinderService
+
+    @Test
+    fun `should get TemporaryCertificates given one exists for the provided details`() {
```

Comment:
or if none exist (nothing specific in function whether one is found or not). (`should get TemporaryCertificates given the provided details`)

Reply:
OK, will do that 👍

Reply:
Done